install/set-up-and-migrate-database.sh

```diff
@@ -1,33 +1,36 @@
 echo "${_group}Setting up / migrating database ..."
 
-# Fixes https://github.com/getsentry/self-hosted/issues/2758, where a migration fails due to indexing issue
-$dc up --wait postgres
+if [[ -z "${SKIP_DB_MIGRATIONS:-}" ]]; then
```

Comment:
Should this be made visible on the `.env` file?

Reply:
Nope. Had to add this purely for CI for now. I think it's more dangerous/harmful to advertise it.